lib/airports.py

```diff
@@ -1,6 +1,7 @@
 import csv
+import os
 
-AIRPORTS_DAT_FILE = '/home/igor/wttrin-geo/share/airports.dat'
+AIRPORTS_DAT_FILE = '{}/airports.dat'.format(os.getenv('WTTR_MYDIR'))
```

Comment:
use inline formatting for python strings, i.e.
`f"{os.getenv('WTTR_MYDIR')}/airports.dat"`

Reply:
I believe they're called "f-strings"

Reply:
Awesome! This has been updated